auditbeat/docs/getting-started.asciidoc

```diff
@@ -220,7 +220,7 @@ sudo service {beatname_lc} start
 ["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
 sudo chown root {beatname_lc}.yml <1>
-sudo ./{beatname_lc} -e -c {beatname_lc}.yml -d "publish"
+sudo ./{beatname_lc} -e -c {beatname_lc}.yml
```

Comment:
I suggest to separate this out into an other PR as it's not related to the change below.

Reply:
Sorry, this commit made it in here by mistake.